prereise/gather/demanddata/nrel_efs/map_states.py

```diff
@@ -52,6 +101,27 @@ def decompose_demand_profile_by_state_to_loadzone(df, save=None):
     # Convert from local hours to UTC time
     df_lz = shift_local_time_by_loadzone_to_utc(df_lz)
 
+    # Create mapping from state abbreviations to loadzone IDs
+    abv2id = {}
+    for k, v in id2abv.items():
+        abv2id.setdefault(v, []).append(k)
+
+    # Determine the loadzones to be inlcuded in the profile, as specified by regions
+    loadzones = set()
+    for region in regions:
+        if len(region) == 2:
+            loadzones = loadzones | set(abv2id[region])
+        else:
+            loadzones = loadzones | interconnect2id[region]
```

Comment:
Not sure whether this works in the same way or will fit in one line.
```python
loadzones = set().union(set(abv2id.get(r, {})) | interconnect2id.get(r, {}) for r in regions)
```

Reply:
Yeah, you are right. There is a work around as following, probably it isn't worth the effort though (I'm getting nerdy here)
```python
loadzones = set().union(*[set(abv2id.get(r, {})) | interconnect2id.get(r, {}) for r in regions]))

or 

from itertools import chain
loadzones = set(chain(*[set(abv2id.get(r, {})) | interconnect2id.get(r, {}) for r in regions]))
```
It's up to you. I'm okay with the current implementation as well.

Reply:
That's really snazzy! I wasn't aware of the star operator. I'll implement this using that method. Thanks!

Reply:
Done.